pkg/health/health.go

```diff
@@ -55,6 +56,14 @@ func Run(bird, bird6, felixReady, felixLive bool, thresholdTime time.Duration) {
 		os.Exit(0)
 	}
 
+	if birdLive {
+		if err := checkBIRDLive(); err != nil {
+			fmt.Printf("calico/node is not ready: Bird/ConfD is not live: %+v", err)
+			os.Exit(1)
+		}
+		os.Exit(0)
```

Comment:
The Exit(0) above this (l56) will stop people from being able to check both Felix and BIRD readiness.  The flow through this function will need tweaking to allow (at least) any combination of liveness checks.

Reply:
Sounds good! Thanks for spotting this!